src/aws.ts

```diff
@@ -308,6 +314,64 @@ export namespace $AWS {
       };
     }
   }
+
+  export namespace Lambda {
+    /**
+     * @param input
+     * @see https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html
+     */
+    // @ts-ignore
+    export function Invoke<Input, Output>(input: {
+      FunctionName: Function<Input, Output>;
+      Payload: Input;
+      ClientContext?: string;
+      InvocationType?: "Event" | "RequestResponse" | "DryRun";
+      LogType?: "None" | "Tail";
+      Qualifier?: string;
+    }): Omit<AWS.Lambda.InvocationResponse, "payload"> & {
+      Payload: Output;
+    };
+
+    export function Invoke(call: CallExpr, context: CallContext): any {
+      const input = call.args[0].expr;
+      if (input === undefined) {
+        throw new Error(`missing argument 'input'`);
+      } else if (input.kind !== "ObjectLiteralExpr") {
+        throw new Error(`argument 'input' must be an ObjectLiteralExpr`);
+      }
+      const functionName = input.getProperty("FunctionName")?.expr;
+      if (functionName === undefined) {
+        throw new Error(`missing required property 'FunctionName'`);
+      } else if (functionName.kind !== "ReferenceExpr") {
+        throw new Error(
+          `property 'FunctionName' must reference a functionless.Function`
+        );
+      }
+      const functionRef = functionName.ref();
+      if (functionRef.kind !== "Function") {
+        throw new Error(
+          `property 'FunctionName' must reference a functionless.Function`
+        );
+      }
+      const payload = input.getProperty("Payload")?.expr;
+      if (payload === undefined) {
+        throw new Error(`missing property 'payload'`);
+      }
+
+      if (isASL(context)) {
+        const task: Partial<Task> = {
+          Type: "Task",
+          Resource: "arn:aws:states:::lambda:invoke",
+          Parameters: {
+            FunctionName: functionRef.resource.functionName,
```

Comment:
missing the result path to get the result payload?

Reply:
ohh, sorry, I didn't realize this was different that the integration in `function.ts`